packages/trpc/server/routers/viewer.tsx

```diff
@@ -159,7 +159,21 @@ const publicViewerRouter = router({
     }),
   // REVIEW: This router is part of both the public and private viewer router?
   slots: slotsRouter,
-  cityTimezones: publicProcedure.query(() => cityMapping),
+  cityTimezones: publicProcedure.query(() => {
+    const maxPops: { [key: string]: { city: string; timezone: string; pop: number } } = {};
+    cityMapping.forEach((city) => {
+      const cityPop = city.pop;
```

Comment:
Let's try being more explicit here:

```suggestion
      const cityPopulationCount = city.pop;
```